app/resources/static/fixed/forms.js

```diff
@@ -361,6 +361,22 @@ function validate_fields() {
     }
   }
 
+  // Validate age format
+  var age_field = $('age');
+  var age_value = age_field.value;
+
+  if (age_value != "") {
+    var curated_age_value = age_value.replace(/\s/g,'');
+    var valid_age_re = /^(\d+-?)+\d+$/;
```

Comment:
I think the question mark should go on the outside of the parens, and the plus sign there now can be dropped, like this:
`^(\d+-)?\d+$`